docs/windows/deployment/overview.md

```diff
@@ -112,13 +113,17 @@ The `<GenerateAppxPackageOnBuild>` set to `true` packages the app and signs it w
 To publish your app, open the **Developer Command Prompt for VS 2022 Preview** terminal and navigate to the project's folder. Run `dotnet` in publish mode:  
 
 ```console
-dotnet publish -f net6.0-windows10.0.19041.0 -c Release
+dotnet publish MyMauiApp.csproj -f net6.0-windows10.0.19041.0 -c Release
 ```
 
+> [!NOTE]
+> When running the `dotnet publish` command without specifying the `.csproj` file, it will start publishing the solution (the `.sln` file) which is unsupported.
+
```

Comment:
Is it the case that the scope of which projects will be published is determined by the current folder, rather than the .sln?

Reply:
@davidbritch did you see his question about scope of published projects?